modules/nlu/src/backend/pipelines/slots/crf_extractor.ts

```diff
@@ -19,26 +18,17 @@ const CRF_TRAINER_PARAMS = {
   'feature.possible_transitions': '1',
   'feature.possible_states': '1'
 }
-
-type Features = Partial<{
-  b_cluster: number
-  b_upper: boolean
-  b_title: boolean
-  b_low: string
-
-  a_cluster: number
-  a_upper: boolean
-  a_low: string
-  a_title: boolean
-
-  w_upper: boolean
-  w_low: string
-  w_title: boolean
-  w_digit: boolean
-  w_bias: number
-  w_eos: boolean
-  w_bos: boolean
-}>
+const FT_PARAMS = {
```

Comment:
`FT_PARAMS: FastTextTrainArgs = { ... }` (~~as FastTextTrainArgs~~) to benefit from typings